resource/parse_jwt.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types=1);
+
+namespace MyApp;
+
+require 'vendor/autoload.php';
+
+use DateTimeImmutable;
+use Lcobucci\Clock\FrozenClock;
+use Lcobucci\JWT\JwtFacade;
+use Lcobucci\JWT\Signer\Hmac\Sha256;
+use Lcobucci\JWT\Signer\Key\InMemory;
+use Lcobucci\JWT\Validation\Constraint;
+
+use function var_dump;
+
+$jwt = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2NTg2OTYwNTIsIm5iZiI6MT'
```

Comment:
Are these values secret? Also it looks like the key we're using in this is different than the one we use in create_jwt.php, is that okay?

Reply:
I forgot to remove that file. It was just there for testing.